README.md

```diff
@@ -27,9 +27,9 @@ ADDED VALUE
 
 Unit Tests:
     - In the same directory used to run the initial program run the unit tests with the following command:
-        python -m unittest -v unittests/test_utils.py   
+        python -m unittest
```

Comment:
Should we not have commands to run each file of unit tests separately too?

Reply:
I can add those commands to the readme

Reply:
yeah, just for the sake of completeness